roseus/test/param-test.l

```diff
@@ -173,6 +173,24 @@
              (bench2 (dotimes (i 100) (ros::get-param "test"))))
           "(ros::get-param-cached test) has less performance than (ros::get-param test)"))
 
+(deftest test-list-param ()
+  (let (ret)
```

Comment:
Forgetting to set _ret_ to `(ros::list-param)`.
Also, seems to fail due to `test2` params set on other tests.  Maybe move this test up or edit the other ones to delete params after created?

Reply:
thank you fixed.